lib/datadog/tracing/distributed/trace_context.rb

```diff
@@ -54,6 +56,9 @@ def extract(data)
             end
           end
 
+          tags ||= {}
+          tags[Tracing::Metadata::Ext::Distributed::TAG_DD_PARENT_ID] = tp_parent_id || '0000000000000000'
+
```

Comment:
This may be me not understanding Ruby code very well or the overall flow here, but won't this always be creating a `_dd.parent_id` tag and adding it the span below (`TraceDigest.new`)?

I can't really follow if this is won't do that if the `datadog` headers are present - I'd imagine that it is correct behavior as that is what the system tests assert, I just can't tell where that is here.

Reply:
I believe this tag will be set if the `tracontext` headers are being and used in the local context. It does not take `datadog` headers in to account. Should we save this work for phase 3?